shared/cmd/helpers.go

```diff
@@ -74,24 +75,27 @@ func EnterPassword(confirmPassword bool, pr PasswordReader) (string, error) {
 }
 
 // ExpandWeb3EndpointIfFile expands the path for --http-web3provider if specified as a file.
-func ExpandWeb3EndpointIfFile(ctx *cli.Context, flag *cli.StringFlag) error {
+func ExpandWeb3EndpointIfFile(ctx *cli.Context, flags *cli.StringSliceFlag) error {
 	// Return early if no flag value is set.
-	if !ctx.IsSet(flag.Name) {
+	if !ctx.IsSet(flags.Name) {
 		return nil
 	}
-	web3endpoint := ctx.String(flag.Name)
-	switch {
-	case strings.HasPrefix(web3endpoint, "http://"):
-	case strings.HasPrefix(web3endpoint, "https://"):
-	case strings.HasPrefix(web3endpoint, "ws://"):
-	case strings.HasPrefix(web3endpoint, "wss://"):
-	default:
-		web3endpoint, err := fileutil.ExpandPath(ctx.String(flag.Name))
-		if err != nil {
-			return errors.Wrapf(err, "could not expand path for %s", web3endpoint)
-		}
-		if err := ctx.Set(flag.Name, web3endpoint); err != nil {
-			return errors.Wrapf(err, "could not set %s to %s", flag.Name, web3endpoint)
+	rawFlags := sliceutil.SplitCommaSeparated(ctx.StringSlice(flags.Name))
```

Comment:
i think bootstrap nodes example uses a multiple flag and doesn't use SplitCommaSeparated	`c.BootstrapNodes = cliCtx.StringSlice(cmd.BootstrapNode.Name)`